### ISSUE TITLE
No way to show FieldArray validation errors for user if FieldArray has no items inside

### ISSUE THREAD

**Issue by nktssh** [erikras/redux-form#2527]
[Release v6.5.0](https://github.com/erikras/redux-form/releases/tag/v6.5.0) introduces bugfix:
```
Do not set FieldArray as touched on submit.
```
Related links #2473 #2014

Here is `meta` props passed to my component (wrapped with `FieldArray`) initially:
```
{
	dirty: false,
	error: "Array should not be empty",
	invalid: true,
	pristine: true,
	submitting: undefined,
	touched: false,
	valid: false,
	warning: undefined
}
```

After user hit "Submit" button, component doesn't rerenders and those props doesn't changed.
Now it's impossible to show error to user if there is no items inside array - previously it was like this:
`(!pristine || touched) && invalid && error`.
Now this always false until user did not interact with component wrapped in `FieldArray`.

Example of this can be found in redux-form [v6.5.0 FieldArray example](http://redux-form.com/6.5.0/examples/fieldArrays/) - just hit "Submit" and you will not see errors regarding to `members`.

Possibly related issue: #2207

Is there any way to show errors to user until he interact with this component?

**Comment by nealoke** [comment]
@nktssh Looking at the example, when I push the submit button without entering anything in the member or club name I get errors.

Might be handy to check the [selector docs](http://redux-form.com/6.5.0/examples/selectingFormValues/) if you want to see if there are any in the array.

**Comment by nktssh** [comment]
@nealoke 
> Looking at the example, when I push the submit button without entering anything in the member or club name I get errors.

that's what I'm talking about - we can't see error for member, however there is one!
And we can't submit form even if we fill "Club Name" field. And it's odd since we can't understand in what reason I can't submit a form

**Comment by JRigotti** [comment]
Removing the `touched` property might solve some bugs, but introduces a new one. As in [this example](http://redux-form.com/6.5.0/examples/fieldArrays/) where you'll never see the `"At least one member must be entered"`.

I came here because I had the following problem: I have a radiogroup wrapped in FieldArray where one of the values of the group have to be selected by the user. Since the `touched` property was removed I can't point to the user that the at least one of those radio buttons must be chosen. 

My "solution" was to regress to `6.4.3` where this still works, but I hope this might be solved in future releases.
